### DISCUSSION TITLE
(ediscovery) Is that any api endpoint available for github legalhold

### DISCUSSION THREAD

**Discussion by dsigninfinnium** [community/community#103123] [Enterprise]
### Select Topic Area

Question

### Body

badically i want to apply legalhold for github user using api , so if there is an endpoint available please provide link of that document .

Thanks,
Jenil Ramani

**Comment by ghostinhershell** [comment]
Hi @dsigninfinnium, 

Thanks so much for reaching out in our community. We do have a support article that walks you through the process of placing a legal hold: [Placing a legal hold on a user or organization](https://docs.github.com/en/enterprise-server@3.9/admin/managing-accounts-and-repositories/managing-users-in-your-enterprise/placing-a-legal-hold-on-a-user-or-organization). It doesn't look like there's an API endpoint available for that. I also want to note that this must be done from an administrative account on GitHub Enterprise Server. 

Let me know if you have any additional questions. Thanks!

**Comment by dsigninfinnium** [comment]
Hi @ghostinhershell ,

okay so currently there is no API endpoint available for that ,so i can't apply hold with API in general is this right ?

Thanks,
Jenil Ramani
